sdk/personalizer/ai-personalizer-rest/review/ai-personalizer.api.md

```diff
@@ -0,0 +1,936 @@
+## API Report File for "@azure-rest/ai-personalizer"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+/// <reference types="node" />
+
+import { Client } from '@azure-rest/core-client';
+import { ClientOptions } from '@azure-rest/core-client';
+import { HttpResponse } from '@azure-rest/core-client';
+import { KeyCredential } from '@azure/core-auth';
+import { RawHttpHeaders } from '@azure/core-rest-pipeline';
+import { RequestParameters } from '@azure-rest/core-client';
+import { StreamableMethod } from '@azure-rest/core-client';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public (undocumented)
+function createClient(Endpoint: string, credentials: TokenCredential | KeyCredential, options?: ClientOptions): PersonalizerClient;
+export default createClient;
```

Comment:
- Do we want to export it for the users?
- Can you add docs for the method?
- If we want to export it for the users, can it have a better name?

Reply:
This is auto-generated code.

Reply:
@HarshaNalluru, I'll check if the lack of docs is Swagger missing it or Codegen missing it.

As for the name, since this is a default export, customers will name it when they import it. 

```ts
import Personalizer from "@azure-rest/ai-personalizer";
```

Since this is a factory function we thought that `createClient` should read fine in the docs.

Reply:
Marking as resolved based on Jose's and my responses.